### PR TITLE
Make Select All Work in Form Editor

### LINKED ISSUE #2411: Select all is not working in forms properties panel
__Describe the Bug__

Executing Cmd+A inside a text input in the forms properties panel does not do anything.

__Steps to Reproduce__

1. Create a new form field
2. Select "field label"
3. Press Cmd + A 
4. See that the content of the input is not selected

__Expected Behavior__

The content of the input is selected completely.

__Environment__

- OS: macOS 11.5
- Camunda Modeler Version: 4.10.0-rc.0

This is not a regression from the latest release, but already discoverable in Camunda Modeler 4.9.0.

### PR DESCRIPTION
**Pull request by philippfromme**
@barmac or @pinussilvestrus please verify the fix on macOS. 👍🏻 

Closes #2411
Related to https://github.com/camunda/camunda-modeler/issues/2541

### CODE REVIEW

**Comment by philippfromme** [comment]
@pinussilvestrus Fabulous! Feel free to approve then. 👍🏻

File: client/src/app/tabs/form/__tests__/FormEditorSpec.js [hunk 1]
```diff
@@ -266,6 +267,26 @@ describe('<FormEditor>', function() {
...
+          // then
+          expect(state.editMenu).to.deep.include(editMenuEntries);
+        };
+
+        const { instance } = await renderEditor(schema, {
+          onChanged: onChangedSpy
+        });
+
+        // when
+        instance.handleChanged();
```
Comment (barmac) [inline client/src/app/tabs/form/__tests__/FormEditorSpec.js:287]: Hmm so the test ends after this call. If `handleChanged` ever gets async, failure in the `onChangeSpy` will only be revealed as an unhandled promise rejection -> ignored by the test framework.

**Review by barmac — APPROVED** [review]

**Comment by philippfromme** [comment]
❤️

File: client/src/app/tabs/form/__tests__/FormEditorSpec.js (hunk 1 above)
Comment (philippfromme) [reply client/src/app/tabs/form/__tests__/FormEditorSpec.js:287]: You're right. All of these tests are probably broken. 🤡 Will fix in a separate PR.

**Pull request merged**